### PR TITLE
Add description to SET_PED_DENSITY_MULTIPLIER_THIS_FRAME

### PR DESCRIPTION
**Pull request by Aloncheeto**
Add comments for better understanding.

### CODE REVIEW

File: PED/SetPedDensityMultiplierThisFrame.md [hunk 1]
```diff
@@ -3,11 +3,15 @@ ns: PED
 ---
 ## SET_PED_DENSITY_MULTIPLIER_THIS_FRAME
 
-```c
+```
```
Comment (AvarianKnight) [inline PED/SetPedDensityMultiplierThisFrame.md:6]: Please don't remove the language tag

File: PED/SetPedDensityMultiplierThisFrame.md [hunk 2]
```diff
@@ -3,11 +3,15 @@ ns: PED
...
 // 0x95E3D6257B166CF2 0x039C82BB
 void SET_PED_DENSITY_MULTIPLIER_THIS_FRAME(float multiplier);
 ```
-
+**Usage:**
+- Call this native every frame
+- Values
+  - 1.0 (maximum) Normal amount of peds in street.
+  - 0.0 (minimum) No peds in the street.
```
Comment (AvarianKnight) [inline PED/SetPedDensityMultiplierThisFrame.md:14]: This should have minimum first.

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

**Comment by ahcenezdh** [comment]
At this point, you can add examples from Lua, JS, or C#

 ```lua
-- 0.0 means no peds, while 1.0 indicates the regular density of peds.
local pedsDensityFactor = 0.0 

Citizen.CreateThread(function()
    while true do
        SetPedDensityMultiplierThisFrame(pedsDensityFactor)
        Citizen.Wait(0)    
    end
end)
```
JavaScript
```js
const pedsDensityFactor = 0.0;

setTick(() => {
    SetPedDensityMultiplierThisFrame(pedsDensityFactor(pedsDensityFactor);  
});
```

and for C# i'm not sure so i'm not going to write it

**Review by 4mmonium — APPROVED** [review]
Approving this PR, thank you all for your contribution 🙂

**Pull request merged**